src/scenes/Activities/scenes/EventRows.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component, Fragment } from 'react';
+import { inject, observer } from 'mobx-react';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+import {
+  TableBody,
+  TableCell,
+  TableRow,
+  withStyles,
+} from '@material-ui/core';
+import cx from 'classnames';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+
+import styles from './styles';
+import TransactionHistoryID from '../../../components/TransactionHistoryAddressAndID/id';
+import TransactionHistoryAddress from '../../../components/TransactionHistoryAddressAndID/address';
+import { getShortLocalDateTimeString } from '../../../helpers/utility';
+import { i18nToUpperCase } from '../../../helpers/i18nUtil';
+import { getTxTypeString } from '../../../helpers/stringUtil';
+
+const EventRows = observer(({ displayedTxs }) => (
```

Comment:
can we add this to the end of the file to follow the same patterns in our other files